docker-compose.nethermind-prysm.yml

```diff
@@ -0,0 +1,120 @@
+version: "3.8"
+
+services:
+  nethermind:
+    image: nethermind/nethermind:$NETHERMIND_VERSION
+    container_name: nethermind
+    volumes:
+      - $EXECUTION_DATA_VOLUME:/execution_data
+      - $CONFIGS_VOLUME:/configs
+    restart: unless-stopped
+    stop_signal: SIGINT
+    stop_grace_period: 2m
+    command: >
+      --config=none.cfg
+      --Init.ChainSpecPath=/configs/chainspec.json
+      --Init.GenesisHash=$GENESIS_HASH
+      --Init.BaseDbPath=/execution_data
+      --Sync.SnapSync=true
+      --JsonRpc.Host=localhost
+      --JsonRpc.Port=8545
+      --JsonRpc.EnginePort=8551
+      --JsonRpc.Enabled=true
+      --JsonRpc.EnabledModules=["eth","web3","net","engine"]
+      --JsonRpc.JwtSecretFile=/configs/jwt.hex
+      --Discovery.Bootnodes=$EXECUTION_BOOTSTRAP_NODE_1,$EXECUTION_BOOTSTRAP_NODE_2
+      --Merge.Enabled=true
+      --KeyStore.KeyStoreDirectory=/execution_data/keystore
+      --Network.MaxActivePeers=50
+
+    network_mode: host
+    logging:
+      driver: "local"
+      options:
+        max-size: "100m"
+
+  prysm_beacon:
+    image: prysmaticlabs/prysm-beacon-chain:$PRYSM_BEACON_VERSION
+    container_name: prysm_beacon
+    depends_on:
+      - nethermind
+    volumes:
+      - $CONSENSUS_DATA_VOLUME:/consensus_data
+      - $CONFIGS_VOLUME:/configs
+    restart: unless-stopped
+    stop_signal: SIGINT
+    stop_grace_period: 2m
+    command: >
+      --accept-terms-of-use
+      --genesis-state=/configs/genesis.ssz
+      --chain-config-file=/configs/config.yaml
+      --datadir=/consensus_data
+      --bootstrap-node $CONSENSUS_BOOTSTRAP_NODE_1,$CONSENSUS_BOOTSTRAP_NODE_2
+      --execution-endpoint=http://localhost:8551
+      --jwt-secret=/configs/jwt.hex
+      --suggested-fee-recipient $FEE_RECIPIENT
+      --min-sync-peers=0
+      --monitoring-host 0.0.0.0
+      --grpc-gateway-host 0.0.0.0
+      --rpc-host 0.0.0.0
+      --verbosity $PRYSM_VERBOSITY
+      --p2p-max-peers 70
+      --subscribe-all-subnets
+      --minimum-peers-per-subnet 0
+      --contract-deployment-block 0
+      --block-batch-limit 512
+      --block-batch-limit-burst-factor 10
+      --p2p-host-ip $EXTERNAL_IP
+      --checkpoint-sync-url=$CHECKPOINT_SYNC_URL
```

Comment:
We don't need `--genesis-beacon-api-url` in Prysm v5+?

Reply:
We don't - what would pair with checkpoint flag is `--weak-subjectivity-checkpoint`, but this is only to ensure the additional safety measures to limit trust, plus it needs to be in format of current block root + epoch, so I think that as long as we are using our own checkpoint solution (checkpointz) then leaving simply this flag is fine

Reply:
Makes sense to me. As long as `genesis-beacon-api-url` is not required for checkpointz sync in Prysm v5+